truenas/webui
# NAS-132749: Update container instance after editing

**Testing:**

1. Add container
2. Edit container details for example by adding an env variable.

**Expected result:** UI listens to an event from middleware and container details card is updated.
